.github/workflows/test-runner-resiliency-test.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Test Runner Resiliency Test
+
+on:
+  push
+
+jobs:
+  test-runner-resiliency:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:12.3
```

Comment:
I think we just upgraded or are about to upgrade to 13

Reply:
See #258

Reply:
Updated postgres image version.